controller/api/models.py

```diff
@@ -643,7 +643,8 @@ def create(self):
         kwargs = {'memory': self.release.config.memory,
                   'cpu': self.release.config.cpu,
                   'tags': self.release.config.tags,
-                  'values': self.release.config.values}
+                  'values': self.release.config.values,
+                  'aname': self.app.id}
```

Comment:
Can we make `aname` something more descriptive? `app_id`, `app_name`, or something similar.